ambry-api/src/main/java/com.github.ambry/config/RouterConfig.java

```diff
@@ -54,6 +54,22 @@
   @Default("2")
   public final int routerScalingUnitMaxConnectionsPerPortSsl;
 
+  /**
+   * The percentage of {@link RouterConfig#routerScalingUnitMaxConnectionsPerPortSsl} or
+   * {@link RouterConfig#routerScalingUnitMaxConnectionsPerPortPlainText} to warm up in the startup.
+   * {@link RouterConfig#routerConnectionsWarmUpTimeoutMs} may need to be adjusted.
+   */
+  @Config("router.connections.warm.up.fraction.per.port")
```

Comment:
Property should be called "percentage".  Fraction would be 0.25, not 25.

Reply:
Will change. I was confused with `Fraction ` and `Percentage `.